livekit-plugins/livekit-plugins-elevenlabs/livekit/plugins/elevenlabs/tts.py

```diff
@@ -292,6 +279,12 @@ async def _run(self) -> None:
             "voice_settings": voice_settings,
         }
 
+        decoder = utils.codecs.AudioStreamDecoder(
```

Comment:
very cool that you don't even need to specify the codec

Reply:
yeah, libav handles everything